lib/hanami/helpers/assets_helper.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Hanami
+  module Helpers
+    module AssetsHelper
+      def javascript(...)
+        _context.assets.javascript(...)
+      end
```

Comment:
Is the intent here to also have a stylesheet helper?

```suggestion
      end
      
      def stylesheet(...)
        _context.assets.stylesheet(...)
      end
```

Reply:
and maybe an image helper?

```suggestion
      end
      
      def image(...)
        _context.assets.image(...)
      end
```

Reply:
We have both of these now :)